packages/smooth_app/lib/pages/product_page.dart

```diff
@@ -254,12 +254,10 @@ class _ProductPageState extends State<ProductPage> {
                 });
                 return;
               case 4:
-                WcFlutterShare.share(
-                    sharePopupTitle: 'Share',
-                    subject: '${_product.productName} (by openfoodfacts.org)',
-                    text:
-                        'Try this food: https://openfoodfacts.org/product/${_product.barcode}/',
-                    mimeType: 'text/plain');
+                Share.share(
+                  'Try this food: https://openfoodfacts.org/product/${_product.barcode}/',
+                  subject: '${_product.productName} (by openfoodfacts.org)',
+                );
```

Comment:
Is it localized yet ?

Reply:
@teolemon I guess not. Is there a process? When we develop new features, we sometimes have to add new text, in English. Is there a way we could say: "hey translator guys, we need your help?" - or something like that?

Reply:
we should think "natively" about l10n and i18n (eg refrain from coding stuff that only works culturally for one country), and if feature can come localized already, it's best. 
I've started a PR to auto-update the ARB file.

Reply:
@teolemon I understand your point, but I think it's not adapted.
Currently we're developing in all directions, experimenting with the UX/UI, imagining possible solutions that have a fair probability to get rejected quickly. Should we spend time finding the right field name and a couple of translations instead of focusing on the user experience? Won't there be inconsistencies in the localization field names? How should the developer say: "OK guys, I've translated into English, French and Italian, but there are still hundreds of languages waiting?".
I think we need a "how-to" guide for developers on this topic. If your PR can help, so much the better.